pkg/cmd/helm/release/release.go

```diff
@@ -353,6 +353,20 @@ func (o *Options) OCIRegistry(repoURL, chartDir, name string) error {
 	return nil
 }
 
+func OCILoginCommand(repoURL string, o *Options) ([]string, error) {
```

Comment:
Any reason to export this function? I don't see it being used outside this function.

Reply:
No good reason, no, will change it